model/BUILD.bazel

```diff
@@ -223,6 +223,17 @@ java_lite_proto_library(
     deps = [":exploration_proto"],
 )
 
+format_import_proto_library(
+    name = "platform_parameter",
+    src = "src/main/proto/platform_parameter.proto",
+)
+
+java_lite_proto_library(
+    name = "platform_parameter_java_proto_lite",
+    visibility = ["//visibility:public"],
```

Comment:
We probably want this to be `oppia_api_visibility`

Reply:
Oh! will update this in the base PR ( as this change came after rebasing this PR over the branch that it is stacked upon)